doc/source/how-to/code/tests.yml

```diff
@@ -1,47 +1,22 @@
-name: CI
-on:
-  pull_request:
-  push:
-    tags:
-      - "*"
-    branches:
-      - main
-
-env:
-  MAIN_PYTHON_VERSION: 3.9
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
-
-jobs:
-  tests:
-    name: Tests
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [windows-latest, macos-latest, ubuntu-latest]
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
-      fail-fast: false
-
-    steps:
-      - name: Checkout project
-        uses: actions/checkout@v3
-
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v4
-        with:
-          python-version: ${{ matrix.python-version }}
-
-      - name: Installing dependencies
-        run: |
-          python -m pip install --upgrade pip
-          <additional dependencies>
-
-      - name: Run tests
-        run: |
-          <command for running your tests>
-
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v2
-        if: matrix.python-version == ${{ env.MAIN_PYTHON_VERSION }}
+tests:
+  name: "Test library"
+  runs-on: ${{ matrix.os }}
+  strategy:
+     matrix:
+         os: [ubuntu-latest, windows-latest]
+         python-version: ['3.7', '3.8', '3.9', '3.10']
+  steps:
+    - name: "Run pytest"
+      uses: pyansys/actions/tests-pytest@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+        pytest-markers: "-k 'mocked'"
+        pytest-extra-args: "--cov=ansys.<library> --cov-report=term --cov-report=xml:.cov/coverage.xml --cov-report=html:.cov/html"
+
+    - name: "Upload Coverage Results"
```

Comment:
```suggestion
    - name: "Upload coverage results"
```

Reply:
I know the report shown is in initial caps--but here, I'd use lower case to be consistent with other names.